### DISCUSSION TITLE
One-to-One relationhip?

### DISCUSSION THREAD

**Discussion by Shmookoff** [collerek/ormar#607] [Q&A] [answered]
I've searched through docs, GitHub Issues, and Discussions, but haven't really found anything on how to declare One-to-One relationship with ormar. Can someone point me to the solution?

**Comment by collerek** [comment, marked as answer]
It's not yet possible, there is a ticket for it in issues

**Reply by Shmookoff** [reply]
Okay, thanks. Up-voted it.
